css/templates/e-commerce/page.css

```diff
@@ -216,7 +216,8 @@ amp-selector [option][selected] {
 
 @media (--breakpoint-md) {
   .commerce-side-panel {
-    background-color: var(--color-panel);
+    position: sticky;
```

Comment:
This style ensures that the side panel stays fixed as the user scrolls. If sticky is not supported, the panel will scroll with the rest of the page. So it degrades gracefully :)